### ISSUE TITLE
path doesn't seem to be resolved correctly

### ISSUE THREAD

**Issue by yyx990803** [pksunkara/vue-builder-webpack-plugin#2]
I'm getting

```
Error: ENOENT: no such file or directory, scandir '/Users/evan/Vue/vue-templates/test/w2/node_modules/vue-builder-webpack-plugin/Users/evan/Vue/vue-templates/test/w2/src'
```

After passing in an already resolved absolute path to the `path` option. Looks like the path is being appended after the directory this plugin is in.

- macOS 10.12.1
- Node 6.9.1
- Webpack 2.1.0-beta.11

**Comment by thierrymichel** [comment]
Same here…

- macOS 10.12.3
- Node 6.9.4
- Webpack 2.2

**Comment by pksunkara** [comment]
I am so sorry. I don't have notifications enabled for this repo and I completely missed everything. I am going to look into it this week.

**Comment by pksunkara** [comment]
Released 0.2.0
